x/ccv/provider/keeper/grpc_query.go

```diff
@@ -30,3 +30,41 @@ func (k Keeper) ConsumerGenesis(c context.Context, req *types.QueryConsumerGenes
 
 	return &types.QueryConsumerGenesisResponse{GenesisState: gen}, nil
 }
+
+func (k Keeper) ConsumerChains(goCtx context.Context, req *types.QueryConsumerChainsRequest) (*types.QueryConsumerChainsResponse, error) {
```

Comment:
Could we keep query names consistent? Some start with 'Query' but this one doesn't.

Reply:
`Done.`